pages/about.html

```diff
@@ -4,9 +4,20 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <!-- <link href="styles/style.css" rel="stylesheet"> -->
+  <link href="styles/about.css" rel="stylesheet">
+  
   <title>Document</title>
 </head>
+<h1>A little about me</h1>
+<h2>Esther "Hoku" McAngus</h2>
```

Comment:
I would put these two headings into the body tag to express that these elements are part of the body of the page. Right now they are siblings to the body tag.